### PR TITLE
added trash and archive features

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this pr functionality of trash and archive are added.

### CODE REVIEW

File: src/components/noteCard/noteCard.jsx [hunk 1]
```diff
@@ -23,9 +27,11 @@ export const NoteCard = (props) => {
             <div className='notecard-section'>
                 <p className='notecard--date'>Created on 01/02/2020</p>
                 <div className="notecard--icons">
-                    <FiEdit className='note--icon' />
-                    <FiArchive className='note--icon' />
-                    <MdDelete className='note--icon' onClick={() => moveNoteToTrash(noteInfo)} />
+                    <MdDelete className='note--icon' title='trash' onClick={() => transferToTrash(noteInfo)} />
+                    {/* <BsTrash className='note--icon' title='trash' onClick={() => transferToTrash(noteInfo)} /> */}
```
Comment (Kothalkarpriya) [inline src/components/noteCard/noteCard.jsx:31]: If commented code is not required do delete them

File: src/context/notesContext/notesContext.jsx [hunk 2]
```diff
@@ -73,15 +76,17 @@ const NoteProvider = ({ children }) => {
...
-        }
-        catch (error) {
-            console.error('error while deleting note', error);
-        }
-    }
+    // const moveNoteToTrash = async (noteData) => {
+    //     try {
+    //         const response = await transferNoteToTrashService(noteData);
+    //         console.log('from note trash', response);
+    //         setNoteList(response.data.notes);
```
Comment (Kothalkarpriya) [inline src/context/notesContext/notesContext.jsx:83]: here also

**Review by Kothalkarpriya — APPROVED** [review]
The code looks clean and understandable. You can merge

File: src/components/noteCard/noteCard.jsx (hunk 1 above)
Comment (ShreyashMahajan) [reply src/components/noteCard/noteCard.jsx:31]: ok thank yo, i will remove that

**Pull request merged**